packages/wallet/src/cip30.ts

```diff
@@ -92,9 +102,23 @@ export const createWalletApi = (
       throw new ApiError(500, 'Nope');
     }
   },
-  getCollateral: async () => {
-    logger.warn('getCollateral is not implemented');
-    return null;
+  getCollateral: async (params: { amount: Cbor }): Promise<Cbor[] | null> => {
```

Comment:
The spec says `null` is a valid return type, however, `getUtxos` was implemented with `undefined` instead :thinking:

Reply:
Yep seems like a bug with getUtxos